lib/MusicBrainz/Server/Form/Collection.pm

```diff
@@ -78,4 +78,19 @@ sub validate_type_id {
     }
 }
 
+sub validate_collaborators {
+    my $self = shift;
+
+    my @collaborators = $self->field('collaborators')->fields;
+    for my $collaborator (@collaborators) {
+        my $id_field = $collaborator->field('id');
+        my $name_field = $collaborator->field('name');
+        if (defined $name_field->value && !(defined $id_field->value)) {
```

Comment:
It would be nice to try to find editor’s id from the given name before rejecting it.

Reply:
I don't like the idea of automatically selecting the editor, but I did change the messages to account for the editor existing and hopefully make them more useful!